utils/database.ts

```diff
@@ -51,6 +51,9 @@ export class Database {
 
   constructor(mongoUri: string) {
     this.mongo.connectWithUri(mongoUri);
+    if (this.mongo.clientId === null || this.mongo.clientId === undefined) {
+      throw new Error("Could not connect to database.");
```

Comment:
Maybe just !this.mongo.clientId

Reply:
The client id can be 0 I think - so this seems safer.

Reply:
was about to say the same thing, I did some tests and `!this.mongo.clientId` throws even though it shouldn't. Just doing some tests to confirm rn

Reply:
Yup, I modified my lambda function to print the client it and it is indeed `0` on success (I'm guessing it acts a as sort of index when multiple connections are established?)